packages/components/src/internal/components/forms/input/DatePickerInput.tsx

```diff
@@ -169,6 +157,7 @@ class DatePickerInputImpl extends DisableableInput<DatePickerInputProps, DatePic
                             isFormsy: false,
                             inputId: queryColumn.name,
                             addLabelAsterisk,
+                            labelClass: 'control-label col-sm-3 col-xs-12 text-left',
```

Comment:
This is breaking bulk edit date fields (with disable buttons) layout

Reply:
Yeah, I meant to take this out.  Trying to fix it so it works in both cases.